job-server/src/main/resources/application.conf

```diff
@@ -77,4 +99,8 @@ spray.can.server {
 
   # Increase this in order to upload bigger job jars
   parsing.max-content-length = 30m
+}
+
+deploy {
+  manager-start-cmd = "./bin/manager_start.sh"
```

Comment:
Once deployed, this file is in the root folder of the Job Server, not in /bin, so this should probably be:

```
manager-start-cmd = "./manager_start.sh"
```

This is still not optimal to have a relative path however, as it means that the server_start.sh script needs to be called from the folder where the Job Server is, otherwise the same error will occur. So for example it's not possible to start the Job Server with ```/opt/job-server/server_start.sh```.

It might be sufficient for now, as one should be able to override this property in the deployed configuration to set an absolute path to the manager_start.sh if necessary.